recipe/meta.yaml

```diff
@@ -1,55 +1,54 @@
 {% set name = "spatialdata" %}
-{% set version = "0.2.5" %}
-{% set python_min = "3.9" %}
+{% set version = "0.3.0" %}
+{% set python_min = "3.10" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/spatialdata-{{ version }}.tar.gz
-  sha256: 4ed8f30347a524548ce54f5957383dd8165ac27ead1c02c68be322854556d17f
+  sha256: 9215dd5171ac9ac0eed01a265f43092c8bdb760f0b1e994795fa274c5c638611
 
 build:
   noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
 
 requirements:
   host:
     - hatch-vcs
     - hatchling
     - pip
-    - python {{ python_min }}
+    - python {{ python_min }},<3.13
   run:
     - anndata >=0.9.1
     - click
     - dask-core >=2024.4.1,<2025
```

Comment:
```suggestion
    - dask-core >=2024.4.1,<=2024.11.2
```

Reply:
mmh I see upstream is pinned to `<=2024.11.2`, since we know dask is a sensible dependency, I would stick to what's the pin upstream, added here https://github.com/scverse/spatialdata/commit/803a66ed7ca00589f2e6ec1df1db90e2d19546fa

@LucaMarconato do you maybe have insights if this is wanted?